fastlane/Fastfile.swift

```diff
@@ -176,6 +176,24 @@ class Fastfile: LaneFile {
         Match.syncCodeSigning(type: .adHoc, environment: .staging, isForce: true)
     }
 
+    func addDevicesGenerateProfilesLane() {
+        desc("Add device and regenerate profiles with match")
+
+        guard let data = Secret.devices.data(using: .utf8),
+              let devices = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] else {
+            return
```

Comment:
Can it throw some error to make the run fail?

Reply:
I don't think it's possible to throw any error because of `try?` here

Reply:
@phongvhd93 if we mark the func as `throws` the Fastlane won't recognize it's as a lane 🙏